cypress/integration/shoppingAutomation.spec.js

```diff
@@ -0,0 +1,72 @@
+import { func } from "assert-plus"
+import {shopping} from "../support/page_object/shoppingPage"
+
+describe('Shopping Page automation', function() {
+let testData
+
+beforeEach(() => {
+    cy.visit('https://automationteststore.com/')
+})
+this.beforeAll(() => {
+    cy.fixture('testData').then(function (data) {
+        testData = data
+        return testData;
+    })
+})
+
+function chooseCategoryAndAddItemToCart (category, items, position) {
+    cy.get(shopping.navMenu).contains(category).click()
+    cy.contains(items, {force: true}).click({force: true})
+    cy.get(shopping.sortItems).select("Date New > Old")
+    cy.get(shopping.productCart).eq(position).invoke('text').then( itemText => {
+        cy.get(shopping.productCart).eq(position).click()
+        cy.get(shopping.productName).find('span').should('have.text', itemText)
+        cy.get(shopping.itemForm).find('select').children().then(($options) => {
+  const randomOption = Math.floor(Math.random() * $options.length);
```

Comment:
zmienne najlepiej tworzyc na gorze(jako pierwsze linijki) funkcji jesli jest taka mozliwosc 

https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/